model/src/main/proto/profile.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Top level proto used to store last used id and all profiles
```

Comment:
Super nit: please end all comment strings in punctuation for proto structures & fields.